experimental/parser/testdata/parser/method/incomplete.proto.stderr.txt

```diff
@@ -4,6 +4,24 @@ error: missing `(...)` around method return type
 20 |     rpc Bar1(foo.Bar) returns foo.Bar;
    |                               ^^^^^^^ help: replace this with `(foo.Bar)`
 
+error: missing method return type in service method
+  --> testdata/parser/method/incomplete.proto:21:5
+   | 
+21 |     rpc Bar2(foo.Bar);
+   |     ^^^^^^^^^^^^^^^^^^ 
+
+error: missing method parameter list in service method
+  --> testdata/parser/method/incomplete.proto:22:5
+   | 
+22 |     rpc Bar3 returns (foo.Bar);
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+
+error: missing method return type in service method
+  --> testdata/parser/method/incomplete.proto:25:5
+   | 
+25 |     rpc Bar6() returns;
+   |     ^^^^^^^^^^^^^^^^^^^ 
+
```

Comment:
The indicated text seems quite vague. The signature does have the "returns" keyword, so it seems like we could use it towards the end of indicating a more useful part of the method.

For missing return type, but returns keyword is present, could indicate a position right after the keyword. If keyword also absent, could indicate a location right after the inputs, or after the name if those are missing, too. Similarly, missing parameter list could indicate a position right after the name.